Week-1/Assignments/index.css

```diff
@@ -0,0 +1,128 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Lato', sans-serif;
+}
+
+/* styles for layout */
+nav {
+    background-color: rgba(231, 220, 220, 0.899);
+    width: 3%;
+    height: 85vh;
```

Comment:
we shouldn't be writing such specific values to css properties as it leads to unresponsiveness

Reply:
okay will do it